### PR TITLE
SIDE-1915: Add documentation pages for User Aptitude

### PR DESCRIPTION
**Pull request by epiccoleman**

### CODE REVIEW

File: docs/src/markdown-pages/aptitudes/user.createJWT.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+name: "JWT"
+links_js: "jwt"
+---
+Receive a JWT with the currently logged in Olive Helps user's email address in the Subject claim.
```
Comment (richseviora) [inline docs/src/markdown-pages/aptitudes/user.createJWT.md:5]:
```suggestion
Receive a JWT with the currently logged in Olive Helps user's email address in the Subject (`sub`) claim.
```

Comment (richseviora) [inline docs/src/markdown-pages/aptitudes/user.createJWT.md:5]: I know the spec talked about other claims but I'm not sure whether they were all implemented - do we want to talk about what values are in each of them besides `exp`?

**Review by richseviora — CHANGES_REQUESTED** [review]
Minor change

**Comment by epiccoleman** [comment]
Closing this - this documentation was added in [PR 194](https://github.com/open-olive/loop-development-kit/pull/194)

**Pull request closed without merging**